transformations/p1_noun_transformation/requirements.txt

```diff
@@ -0,0 +1,4 @@
+git+git://github.com/pawan2411/wptools@30738f1
```

Comment:
This line is causing the build failure. Please change this to wptools==specific_version or just wptools.